Jellyfin.Server/Migrations/Routines/MoveExtractedFiles.cs

```diff
@@ -68,51 +68,34 @@ public MoveExtractedFiles(
     /// <inheritdoc />
     public void Perform()
     {
-        const int Limit = 500;
+        const int Limit = 5000;
         int itemCount = 0, offset = 0;
 
         var sw = Stopwatch.StartNew();
-        var itemsQuery = new InternalItemsQuery
-        {
-            MediaTypes = [MediaType.Video],
-            SourceTypes = [SourceType.Library],
-            IsVirtualItem = false,
-            IsFolder = false,
-            Limit = Limit,
-            StartIndex = offset,
-            EnableTotalRecordCount = true,
-        };
-
-        var records = _libraryManager.GetItemsResult(itemsQuery).TotalRecordCount;
+
+        using var context = _dbProvider.CreateDbContext();
+        var records = context.BaseItems.Where(b => b.MediaType == MediaType.Video.ToString() && !b.IsVirtualItem && !b.IsFolder).Count();
         _logger.LogInformation("Checking {Count} items for movable extracted files.", records);
 
         // Make sure directories exist
         Directory.CreateDirectory(SubtitleCachePath);
         Directory.CreateDirectory(AttachmentCachePath);
-
-        itemsQuery.EnableTotalRecordCount = false;
         do
         {
-            itemsQuery.StartIndex = offset;
-            var result = _libraryManager.GetItemsResult(itemsQuery);
-
-            var items = result.Items;
+            var items = context.BaseItems.Where(b => b.MediaType == MediaType.Video.ToString() && !b.IsVirtualItem && !b.IsFolder).OrderBy(e => e.Id).Skip(offset).Take(Limit).ToList();
```

Comment:
Wouldn't it be more efficient to join MediaStreamInfos instead of querying once per baseitem?

Reply:
What do you mean by this? We just filter by columns which are in the BaseItemEntity?